src/thunderbots/software/ai/passing/main.cpp

```diff
@@ -0,0 +1,63 @@
+#include <ros/ros.h>
```

Comment:
Now that the `PassGenerator` is a threaded object, I don't think this file is necessary anymore?

Reply:
nope, removed